main/test/flix/Test.Exp.String.flix

```diff
@@ -27,4 +27,10 @@ namespace Test/Exp/String {
     @test
     def testString09(): String = "\\"
 
+    @test
+    def testString10(): String = "\""
+
+    @test
+    def testString11(): String = "\'"
+
```

Comment:
Do we need some `\uXXXX` tests? and maybe in general some more tests with multiple escapes?